server/rest/challenge.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+import cherrypy
+
+from girder.api import access
+from girder.api.describe import Description
+from girder.api.rest import loadmodel
+from girder.constants import AccessType
+from girder.plugins.challenge.rest.challenge import Challenge
```

Comment:
I'm wondering if it is still a net benefit to have challenge be a separate plugin.  Not QA for this PR specifically I realize...